app/services/adapters/structs/crime_application.rb

```diff
@@ -52,22 +52,6 @@ def income
         Structs::IncomeDetails.new(means_details.income_details)
       end
 
```

Comment:
Superfluous now we access payments via income. Removing them from here also to prevent accessing of payments via CrimeApplication.